Description: Upstream changes introduced in version 1.0-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 deepin-software-center (1.0-1) natty; urgency=low
 .
   * Initial release (Closes: #nnnn)  <nnnn is the bug number of your ITP>
 .
 The person named in the Author field signed this changelog entry.
Author: Leng Ganghua <hiweed@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ deepin-software-center-1.0/src/deepin-software-center.desktop
@@ -0,0 +1,15 @@
+[Desktop Entry]
+Name=Deepin Software Center
+Name[zh_CN]=Deepin 软件中心
+Name[zh_TW]=Deepin 軟體中心
+Comment=A general Linux software manager
+Comment[zh_CN]=Linux 通用软件管理器
+Comment[zh_TW]=Linux 通用軟體管理者
+Exec=deepin-software-center
+Icon=deepin-software-center
+Terminal=false
+X-MultipleArgs=false
+Type=Application
+Encoding=UTF-8
+Categories=Application;System;Settings
+StartupNotify=false
--- /dev/null
+++ deepin-software-center-1.0/src/deepin-software-center
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+cd /usr/share/deepin-software-center
+gksu ./deepin-software-center.py
